core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -156,6 +160,14 @@ private SaltConfig createSaltConfig(Stack stack, Cluster cluster, GatewayConfig
             servicePillar.put("smartsense-credentials", new SaltPillarProperties("/smartsense/credentials.sls", smartSenseCredentials));
         }
 
+        if (hiveConfigProvider.isRdsConfigNeedForHiveMetastore(cluster.getBlueprint())) {
+            Map<String, Object> postgres = new HashMap<>();
+            postgres.put("database", hiveConfigProvider.getHiveDb());
```

Comment:
I would move the property name related logic into the HiveConfigProvider, as we discuss it :)